backend/data/src/main/kotlin/io/tolgee/development/testDataBuilder/data/TranslationsTestData.kt

```diff
@@ -221,10 +223,14 @@ class TranslationsTestData {
           addTranslation {
             language = germanLanguage
             text = "I am key $padNum's german translation."
+            if (it < 10)
```

Comment:
I don't like you're editing the `generateLotOfData` which has it's own purpose. You can create your custom method for this test, where you can directly return the tagged keys. Then you wouldn't have to rely on the range 1..9.

Reply:
I used `addFewKeysWithTags` instead which was missing translations, so I added them.

I see it as an equal option to creating brand new test data for this usecase. If you find it better, I can do it like new data.